cpu/nrf52/ldscripts/nrf52832xxaa.ld

```diff
@@ -18,10 +18,14 @@
  * @}
  */
 
+rom_length = 512K;
```

Comment:
This is not quite correct, I think the value of `rom_length` should be dependent on the `boot_offset` -> `rom_length = 512K - boot_offset` or even better: ROM length should be dependent on the slot chosen and on the slot size...